README.md

```diff
@@ -52,11 +52,12 @@ The version of the extension for the Opera browser can be found at the [Opera ad
 To get a local copy of the current code, clone it using git:
 
     $ git clone git://github.com/mozilla/pdf.js.git pdfjs
-    $ cd pdfjs
+    $ cd pdf.js
```

Comment:
If you execute the *actual* `git clone` command above, which explicitly specifies the directory name, the folder will be named `pdfjs`. Please revert this change.